src/nvim/ex_docmd.c

```diff
@@ -9409,9 +9409,20 @@ static void ex_folddo(exarg_T *eap)
 
 static void ex_terminal(exarg_T *eap)
 {
-  char cmd[512];
-  snprintf(cmd, sizeof(cmd), ":enew%s | call termopen('%s') | startinsert",
-      eap->forceit==TRUE ? "!" : "",
-      strcmp((char *)eap->arg, "") ? (char *)eap->arg : (char *)p_sh);
-  do_cmdline_cmd((uint8_t *)cmd);
+  // We will call `termopen()` will `[&shell]` if given a {cmd}.
```

Comment:
The brackets denote that we send it as a list, but using `'shell'` instead of `&shell` does seem to make sense.